README.md

```diff
@@ -53,7 +53,10 @@ You can manually include the Markdown processor by setting the `processor` optio
 
 Each fenced code block inside a Markdown document has a virtual filename appended to the Markdown file's path.
 
-The virtual filename's extension will match the fenced code block's syntax tag, so for example, <code>```js</code> code blocks in <code>README.md</code> would match <code>README.md/*.js</code>.
+The virtual filename's extension will match the fenced code block's syntax tag, except for the following: `javascript`, `ecmascript`, `typescript` and `markdown`.
+The exceptions are mapped to their shorter syntax tag variants: `js`, `js`, `ts` and `md` respectively.
+So for example, <code>```js</code> code blocks in <code>README.md</code> would match <code>README.md/*.js</code> and <code>``typescript</code> in <code>CONTRIBUTING.md</code> would match <code>CONTRIBUTING.md/*.ts</code>.
+
```

Comment:
I think this may be easier to read:
```suggestion
The virtual filename's extension will match the fenced code block's syntax tag, except for the following: 

* `javascript` and `ecmascript` are mapped to `js`
* `typescript` is mapped to `ts`
* `markdown` is mapped to `md`

For example, <code>```js</code> code blocks in `README.md` would match `README.md/*.js` and <code>``typescript</code> in `CONTRIBUTING.md` would match `CONTRIBUTING.md/*.ts`.

```